trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -1453,6 +1466,24 @@ mod tests {
         hex::decode(b_str).unwrap()
     }
 
+    // TODO: Move behind a feature flag just used for debugging (but probably not
```

Comment:
I'm not sure I understand what the plan is here... Move behind a flag but probably not? What is the plan?